onnxruntime/core/framework/utils.cc

```diff
@@ -210,15 +211,25 @@ static common::Status CalculateStaticCopyInfoForFeed(const SessionState& session
                                                      const std::string& input_name,
                                                      MLValueCopyInfo& copy_info) {
   std::vector<SessionState::NodeInfo> node_info_vec;
-  ORT_RETURN_IF_ERROR(session_state.GetInputNodeInfo(input_name, node_info_vec));
-  const auto& node_info = node_info_vec.front();  // all consumers of a feed have the same device so first entry is fine
+  if (session_state.GetInputNodeInfo(input_name, node_info_vec) == Status::OK()) {
```

Comment:
`Failed to find output name in the mapping:  + output_name` vs `Could not find OrtValue with name`

... doesn't seem to change the semantics of the error message much but happy to make changes if you feel otherwise.

---
In reply to: [598980770](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r598980770) [](ancestors = 598980770)

Reply:
Fair enough... as long as nobody has a test that depends on the exact string in the name, seems fine to let it change to that.